examples/with-passport/pages/index.js

```diff
@@ -0,0 +1,36 @@
+import { useUser } from '../lib/hooks'
+import Layout from '../components/layout'
+
+const Home = () => {
+  const user = useUser()
+
+  return (
+    <Layout>
+      <h1>Passport.js Example</h1>
+
+      <p>Steps to test the example:</p>
+
+      <ol>
+        <li>Click Login and enter an username and password.</li>
```

Comment:
On the login screen, there's a way to sign up by clicking "I don't have an account" but it's not part of the instruction, so I got confused.

<img width="368" alt="Screen Shot 2020-03-03 at 9 10 56 AM" src="https://user-images.githubusercontent.com/992008/75801375-e1b80100-5d2f-11ea-87e2-112da70c0665.png">

What if we have two sets instructions:

1. Login → click profile → Logout
2. Signup → click profile → Logout (and add Signup to the menu bar)

Reply:
Because of layout changes in the header I went for the simplest way